tests/scenarios/scenario1/pool-clear/3.check-node-pool.test.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const testUtils = require('../../../../lib/utils/test-utils')
+const utils = require('./utils')
+
+describe('New wallet which was sent transaction', () => {
+  it('should have coins in the wallet', async () => {
```

Comment:
The description is a bit misleading because we do not check if the address has coins.